lib/containers/utils.pm

```diff
@@ -112,7 +112,7 @@ sub basic_container_tests {
     if (script_run("$runtime container exec $container_name zypper -n in curl", 300)) {
         record_info('poo#40958 - curl install failure, try with force-resolution.');
         my $output = script_output("$runtime container exec $container_name zypper in --force-resolution -y -n curl", 600);
-        die('error: curl not installed in the container') unless ($output =~ m/Installing: curl.*done/);
+        die('error: curl not installed in the container') unless (($output =~ m/Installing: curl.*done/) || ($output =~ m/The highest available.*installed/));
```

Comment:
Is it possible to make this specific to `curl`?

Reply:
I can try to parse the first line "'curl' is already installed". I'm just afraid of single quotes

Reply:
You're not alone ;-)

How about something like `curl .* already installed` ?

Reply:
Alright, applied your suggestion